src/main/java/com/redhat/cloud/notifications/GwResource.java

```diff
@@ -285,14 +287,14 @@ private static String buildResponseEntity(boolean success, String details) {
         return response.encode();
     }
 
-    private Optional<X509Certificate> getX509Certificate(String bundle, String app, String certificateDn) {
+    private Optional<SourceEnvironment> getSourceEnvironment(String bundle, String app, String identificationData) {
         try {
-            X509Certificate x509Certificate = restValidationClient.validateCertificate(bundle, app, certificateDn);
-            Log.infof("Certificate validated, coming from source environment %s", x509Certificate.sourceEnvironment);
-            return Optional.of(x509Certificate);
+            SourceEnvironment sourceEnvironment = restValidationClient.validateCertificate(bundle, app, identificationData);
+            Log.infof("Identification validated, coming from source environment %s", sourceEnvironment.name);
```

Comment:
General naming suggestion in this method:
```suggestion
            Log.infof("Authentication validated, coming from source environment %s", sourceEnvironment.name);
```